org.js.nuclear.Nuclear.json

```diff
@@ -31,19 +31,16 @@
                 {
                     "type":"file",
                     "url":"https://raw.githubusercontent.com/advaithm/flathub/nuclear/org.js.nuclear.Nuclear.metainfo.xml",
-                    "sha256":"17f2d8221f3cfec579457c05a4a56b85dcff2dd44bc86d5a2bca85e088af7c39",
                     "path": "org.js.nuclear.Nuclear.metainfo.xml"
                 },
                 {
                   "type":"file",
                   "url":"https://raw.githubusercontent.com/advaithm/flathub/nuclear/org.js.nuclear.Nuclear.desktop",
-                  "sha256":"61256cb50cbfa54d93b9c344f4795f42f7dc73cd195be17915403d0ac281ce88",
                   "path": "org.js.nuclear.Nuclear.desktop"
                 },
                 {
                     "type":"file",
                     "url":"https://github.com/advaithm/flathub/raw/nuclear/512.png",
-                    "sha1":"f7568bc736304246d054faa7cf528c3e8bc0a6d5",
```

Comment:
And here I said to have the icon a local file like the .desktop and .appdata.xml added to this repository remove `url`, and instead specify `path`.

You can remove it when the upstream release has it.

Reply:
oh okay